website/docs/r/sql_database_instance.html.markdown

```diff
@@ -229,6 +229,16 @@ when the resource is configured with a `count`.
 * `settings.version` - Used to make sure changes to the `settings` block are
     atomic.
 
+<a id="timeouts"></a>
```

Comment:
No need for this. The h2 tag generated from `##timeouts` will have its id set to `timeouts`.

Reply:
Removed.